### PR TITLE
devices: implement org_id support.

### PR DESCRIPTION
**Pull request by ldjebran**
Fixes #THEEDGE-1980

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [X] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes
- [X] I run `make lint` to prints out coding style mistakes and fix them
- [X] I run `make test-no-fdo` to check all tests passed (beside fdo)

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
d0cc309c-1b5f-42f7-922c-ea43a8205406
dc0211b7-8c27-401b-8de2-36e92b26acd1

File: pkg/routes/devicegroups.go [hunk 1]
```diff
@@ -636,14 +636,14 @@ func UpdateAllDevicesFromGroup(w http.ResponseWriter, r *http.Request) {
...
 	ctxLog.Info("Updating all devices from group", deviceGroup.ID)
-
-	account, err := common.GetAccount(r)
-	if err != nil {
-		ctxServices.Log.WithFields(log.Fields{
-			"error":   err.Error(),
-			"account": account,
-		}).Error("Error retrieving account")
-		respondWithAPIError(w, ctxServices.Log, errors.NewInternalServerError())
+	account := readAccount(w, r, ctxServices.Log)
```
Comment (AaronH88) [inline pkg/routes/devicegroups.go:639]: As discussed on slack, a slight re-work here :smile:

**Comment by ldjebran** [comment]
close and reopen this PR as ci.int.devshift.net is hanging

**Pull request closed without merging**